kafka-ui-react-app/src/components/Topics/shared/Form/TopicForm.tsx

```diff
@@ -79,6 +79,10 @@ const TopicForm: React.FC<Props> = ({
     navigate(clusterTopicPath(clusterName, topicName));
   };
 
+  const getDynamicSource = (key: string) =>
+    config &&
+    config[key as string]?.source === ConfigSource.DYNAMIC_TOPIC_CONFIG;
```

Comment:
why do we have a casting here

Reply:
Removing `as string` casting, it's not necessary 👌